exec/java-exec/src/main/codegen/templates/Parser.jj

```diff
@@ -7727,6 +7764,8 @@ SqlPostfixOperator PostfixRowOperator() :
 |   < DATETIME_INTERVAL_CODE: "DATETIME_INTERVAL_CODE" >
 |   < DATETIME_INTERVAL_PRECISION: "DATETIME_INTERVAL_PRECISION" >
 |   < DAY: "DAY" >
+|   < DAYOFWEEK: "DAYOFWEEK" >
+|   < DAYOFYEAR: "DAYOFYEAR" >
```

Comment:
Should we add a unit test for these synonyms?

Reply:
Yes we should, thanks, added.